app/views/tutors/find_students.html.erb

```diff
@@ -63,29 +63,32 @@
                                     role="tabpanel"
                                     aria-labelledby="<%=true_class %>-tab">
                                     <% x = Course.where(name: true_class, semester: Course.current_semester).first %>
```

Comment:
Perhaps you could name this variable something more descriptive like ```coruse_obj``` instead. Or put a comment hinting at its meaning